tests/test_integrated_channels/test_integrated_channel/test_exporters/test_content_metadata.py

```diff
@@ -353,3 +353,43 @@ def test_export_fetches_content_only_when_update_needed(self, mock_ent_catalog_a
         # We should make a call for the enterprise catalog, but no call for the metadata because no update needed
         assert mock_ent_catalog_api.return_value.get_content_metadata.call_count == 0
         assert mock_ent_catalog_api.return_value.get_catalog_diff.call_count == 1
+
+    def test__check_matched_content_updated_at_incomplete_transmission(self):
+        """
+        Test the __check_matched_content_updated_at function when the transmission is incomplete.
+        """
+        past_transmission = factories.ContentMetadataItemTransmissionFactory(
```

Comment:
Since this generated transmission audit item has a `remote_updated_at` value that comes **after** your `mock_matched_items` updated at values, it shouldn't be found by `_check_matched_content_updated_at`. I'd love to have an additional audit item (or a separate test that has an audit item!) that _does_ get found because of it's update time (and not because it has a `marked_for` value. That way we'd know the `_check_matched_content_updated_at` method:
1) finds items that have a `marked_value`
2) finds items that have update times that come before the `matched_items` update time
3) does not find items that do not meet either of these criteria

Reply:
added a 3rd transmission, following the 3 points above we would only find 2 out of the 3 transmissions:

`past_transmission_updated_at_before_mock` and `past_transmission_updated_at_marked_for`